src/main/java/com/networknt/schema/BaseJsonValidator.java

```diff
@@ -242,6 +166,19 @@ public JsonNode getSchemaNode() {
         return this.schemaNode;
     }
 
+    public JsonNode getDefaultNode(JsonSchema schema) {
```

Comment:
Yes, I do agree with you. However, the PropertiesValidator, PrefixItemsValidator, ItemsValidator202012, and ItemsValidator class extend the BaseJsonValidator so it is better to define getDefaultNode in BaseJsonValidator only because in future if sub classes require getDefaultNode method then it can be used. Should I raise merge request for getDefaultNode?

Reply:
You can raise a pull request to move this to a utility class. This method doesn't use any class properties and BaseJsonValidator has enough methods as it is. It's also not really a method that has general usage other than the classes it is already in.